Documentation/libibverbs.md

```diff
@@ -56,3 +56,15 @@ need to increase this limit.  This is usually done for ordinary users
 via the file /etc/security/limits.conf.  More configuration may be
 necessary if you are logging in via OpenSSH and your sshd is
 configured to use privilege separation.
+
+# Debugging
+
+### Enabling debug prints
+
+Library and providers debug prints can be enabled using the `VERBS_LOG_LEVEL`
+environment variable, the output shall be written to the file provided in the
+`VERBS_LOG_FILE` environment variable. When the library is compiled in debug
+mode and no file is provided the output will be written to stderr.
```

Comment:
I'm not sure that this is fully correct.

Can you please write it as a table for every build mode?
What is expected if variables set/not set and regular prints vs. datapath.

It will be clearer for everyone.

BTW, we will remove our MLX5 debug variables without any need in legacy wrapper.

Thanks

Reply:
Thanks, added a table.